README.md

```diff
@@ -41,6 +41,27 @@ http.get('http://example.com', response => {
 });
 ```
 
+### Using DNS-over-HTTPS resolution
+
+```js
+const CacheableLookup = require('cacheable-lookup');
+const DoHResolver = require('doh-resolver');
+const https = require('https');
+
+const resolver = new DoHResolver({
+  // Your DNS-over-HTTPS servers to target
+  servers: ['1.1.1.1', '8.8.8.8'],
+  // Under any networking isssue, use OS lookup as fallback
+  onError: cb => cb(null, [])
```

Comment:
`onError` is called with `(cb, error)`.

Since `doh-resolver` performs an HTTP request, if an error happened for whatever reason, then just resolve the callback with empty results to enable resolving via OS lookup

https://github.com/szmarczak/cacheable-lookup/blob/master/source/index.js#L339=